lib/cassandra/cluster/schema/cql_type_parser.rb

```diff
@@ -72,6 +72,9 @@ def lookup_type(node, types)
             Cassandra::Types.tuple(*node.children.map { |t| lookup_type(t, types)})
           when 'empty'             then
             Cassandra::Types.custom('org.apache.cassandra.db.marshal.EmptyType')
+          when /^'/ then
```

Comment:
even better, is to use `if name.start_with?('\'')` but that'll change the case statement